### DISCUSSION TITLE
not able to apply Student developer pack

### DISCUSSION THREAD

**Discussion by MangalNathYadav** [community/community#69380] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/68454356/5a45a4b5-cf2d-4cfb-a151-961ea68b9063)
![image](https://github.com/community/community/assets/68454356/c5cc8318-09fe-4cd5-b461-5e35cd3f52a8)

i am trying to apply but this comes. i dont know what to do now . i applied 2 years before and it was accepted but that pack expired so i am applyiing again. but this comes

**Comment by arhamansari11** [comment]
I think student developer pakage is close now a days!
